Corona-Warn-App/src/main/res/values/styles.xml

```diff
@@ -140,6 +140,23 @@
         <item name="android:textColor">@color/colorStableLight</item>
     </style>
 
+    <style name="greenButtonNotPressed">
```

Comment:
why is this button called green if you use `colorSurface2`?

Reply:
states don't have to be handled over different stylings. If none of the existing color state lists have this behaviour create a new one and set the button to disabled instead only changing the color.